src/js/common/components/SortableTable.jsx

```diff
@@ -33,6 +33,14 @@ class SortableTable extends React.Component {
       }
     }
 
+    if (field.nonSortable) {
+      return (
+        <th key={field.value}>
+          {field.label}
+        </th>
+      );
+    }
```

Comment:
I'd like to see this at the top of this function so it's clearly separate and not in the midst of the logic that deals with sorting stuff.